konfera/urls.py

```diff
@@ -15,9 +15,13 @@
 """
 from django.conf.urls import url, include
 from konfera import views
+from konfera.models import event
 
 urlpatterns = [
     url(r'^events/', include('konfera.event.urls'), name='event_list'),
```

Comment:
I know you havent add name here, but can you please remove it, we will have nice clean urls.py. Thanks.